newt/resolve/resolve.go

```diff
@@ -987,6 +994,22 @@ func (r *Resolver) resolveDepsAndCfg() error {
 	r.sysinitCfg = sysinit.Read(lpkgs, &r.cfg)
 	r.sysdownCfg = sysdown.Read(lpkgs, &r.cfg)
 
+	r.preBuildCmdCfg = extcmd.Read("pre_build_cmds", lpkgs, &r.cfg,
+		func(lpkg *pkg.LocalPackage,
+			settings map[string]string) map[string]string {
+			return lpkg.PreBuildCmds(settings)
+		})
+	r.preLinkCmdCfg = extcmd.Read("pre_link_cmds", lpkgs, &r.cfg,
+		func(lpkg *pkg.LocalPackage,
+			settings map[string]string) map[string]string {
+			return lpkg.PreLinkCmds(settings)
+		})
+	r.postBuildCmdCfg = extcmd.Read("post_build_cmds", lpkgs, &r.cfg,
```

Comment:
After reading the code, I feel like this should be 'post_link_cmds' rather than 'post_build_cmds'. I think that would better communicate the order in which scripts are executed. First glance I expected order to be pre_build -> post_build -> pre_link, and I was wondering what happened with the post_link step :).

Reply:
Thanks @mkiiskila.  For some reason I have a preference for `post_build`, but I agree there is some ambiguity there, so I did the right thing and renamed it to `post_link` :).